markets/perps-market/storage.dump.json

```diff
@@ -228,46 +228,53 @@
           "slot": "6",
           "offset": 0
         },
+        {
+          "type": "uint256",
+          "name": "currentLiquidationReward",
```

Comment:
What are these storage slots changes?

Reply:
Sorry my local setup was messed up and produced these. Should be removed